packages/experimental/Avatar/src/JSAvatar.helpers.ts

```diff
@@ -59,45 +43,23 @@ export function convertCoinColorFluent(coinColor: AvatarColor): string {
   return colorTableFluent[coinColor];
 }
 
-export function calculateEffectiveSizes(tokens: JSAvatarTokens): AvatarSizeConfig {
-  const { size, avatarSize, iconSize, iconStrokeWidth, initialsSize } = tokens;
-
-  if (size) {
-    return sizeTable[size];
-  } else {
-    const {
-      physicalSize: defaultPhysicalSize,
-      iconSize: defaultIconSize,
-      iconStrokeWidth: defaultIconStrokeWidth,
-      initialsSize: defaultInitialsSize,
-    } = sizeTable['size40'];
-
-    return {
-      physicalSize: avatarSize || defaultPhysicalSize,
-      iconSize: iconSize || defaultIconSize,
-      iconStrokeWidth: iconStrokeWidth || defaultIconStrokeWidth,
-      initialsSize: initialsSize || defaultInitialsSize,
-    };
-  }
-}
-
-export function getRingConfig(size: number): any {
+export function getRingConfig(size: string | number): any {
   if (size <= 48) {
```

Comment:
Can you compare a string to a number? This seems unsafe, I think there should be additional typechecking here before doing this comparison and the size as number conversions

Reply:
I feel like this function should just take a number, and there should be some defaults we set in the styling file if the width is entered as a string. That seems like a cleaner way to handle this. Alternatively, maybe we fix width and height to just be number instead of taking the same type of width/heigh as ViewStyle.

Reply:
yeah, that wasn't a good solution. Used width/height as number instead of ViewStyle